README.md

```diff
@@ -1,5 +1,7 @@
 # WarsawJS Workshop: Wstępna konfiguracja
 
+**Domyślny config dostępny jest w pliku `index.md` lub na https://warsawjs.github.io/workshop-setup/**
```

Comment:
Bare URL

Reply:
Ło panie całe życie w błędzie. Dopiero teraz zwróciłem uwagę, że bare url to jest `<example.com>`, a nie po prostu `example.com`.